### PR TITLE
Midstream CI changes for release 0.13

### PR DESCRIPTION
**Pull request by aliok**
Same as https://github.com/openshift-knative/eventing-operator/pull/2

Procedure:
1. Push upstream 0.13 branch to this repo as is
2. Delete OWNERS, create Dockerfile
3. Push to own repo, create a PR against the *release-0.13* branch

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RUN cp -Lr ${GOPATH}/src/knative.dev/eventing-operator/cmd/manager/kodata /tmp
+
+FROM openshift/origin-base
+COPY --from=builder /tmp/manager /ko-app/eventing-operator
+COPY --from=builder /tmp/kodata/ /var/run/ko
+ENV KO_DATA_PATH="/var/run/ko"
+LABEL \
+    com.redhat.component="openshift-serverless-1-tech-preview-knative-eventing-rhel8-operator-container" \
+    name="openshift-serverless-1-tech-preview/knative-eventing-rhel8-operator" \
+    version="v0.13.0" \
```
Comment (matzew) [inline Dockerfile:15]: is that the proper version ?

Comment (aliok) [reply Dockerfile:15]: @matzew Took it from https://github.com/openshift-knative/serving-operator/commit/fa90677d2016f6593eb6ab116c232fbcdb611ee6

Comment (aliok) [reply Dockerfile:15]: cc @markusthoemmes

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM openshift/origin-release:golang-1.13 AS builder
```
Comment (matzew) [inline Dockerfile]: we have 

`FROM rhel8/go-toolset:1.13.4 AS builder` in our downstream builds.

and @rhuss has that in here too:
https://github.com/openshift/knative-client/blob/master/Dockerfile.rhel#L1

Comment (aliok) [reply Dockerfile]: updated to that

File: Dockerfile (hunk 1 above)
Comment (matzew) [reply Dockerfile:15]: I mainly asked b/c eventing itself has now 0.13.3 :-) 

but this version seems to be ok...

/lgtm

**Comment by aliok** [comment]
That image failed here:
https://github.com/openshift/release/pull/7812

```
docke...hel8/go-toolset not found: does not exist or no pull access
```

Gonna revert to old image

**Comment by aliok** [comment]
Created https://github.com/openshift-knative/eventing-operator/pull/4

**Pull request merged**